suites/quincy/rados/tier-2_rados_test_bluestore.yaml

```diff
@@ -0,0 +1,125 @@
+# Suite contains basic tier-2 rados tests
```

Comment:
Please add the suite for pacific as well.

Reply:
Added the pacific suite